rackcity/api/objects/rack_range.py

```diff
@@ -1,11 +1,11 @@
 from rest_framework import serializers
 from rackcity.models.rack import validate_row_letter
-from rackcity.models import Datacenter
+from rackcity.models import Site
 
 
 class RackRangeSerializer(serializers.Serializer):
     datacenter = serializers.PrimaryKeyRelatedField(
-        label="datacenter", queryset=Datacenter.objects.all(), required=False
+        label="datacenter", queryset=Site.objects.all(), required=False
```

Comment:
are you waiting to make changes to rack range serializer?

Reply:
TBD on that field, might actually keep it datacenter because a rack will always have a datacenter never an offline storage site